README.md

```diff
@@ -11,45 +17,45 @@ Note that ruby 1.8 support was dropped in ruby-oci8 2.2.0.
 Use ruby-oci8 2.1.8 for ruby 1.8.
 
 What's new
-==========
+----------
 
 See {file:NEWS}.
 
 Sample one-liner
-================
+----------------
 
 Connect to scott/tiger, select `emp` and print as CSV format.
 
     ruby -r oci8 -e "OCI8.new('scott', 'tiger').exec('select * from emp') do |r| puts r.join(','); end"
 
 Homepage
-========
+--------
 
 * http://www.rubydoc.info/github/kubo/ruby-oci8
 
 Installation
-============
+------------
```

Comment:
I think instead of `--------` we can use `## Installation` is more suitable for change in README.md file.
May be `##` is better option.
Output -->
![screen shot 2016-12-28 at 11 57 10 pm](https://cloud.githubusercontent.com/assets/1284716/21528551/646bd212-cd59-11e6-8c04-62fbc27dee66.png)